server/src/main/java/com/lunchtime/models/BaseEntity.java

```diff
@@ -0,0 +1,37 @@
+package com.lunchtime.models;
+
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import javax.persistence.Column;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import java.util.Date;
+
+
+
+@MappedSuperclass
+@Data
+public class BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
You should change GenerationType.IDENTITY to GenerationType.AUTO becouse IDENTITY used with MySQL DB.